### PR TITLE
Correct spelling of activities

### PR DESCRIPTION
**Pull request by rememberlenny**
## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

**Comment by cretz** [comment]
Thanks! I can't merge because the CLA isn't signed, but no biggie, I can include this typo fix in my next samples work.

**Review by flossypurse — APPROVED** [review]

**Comment by robholland** [comment]
@rememberlenny We can't merge this unless the CLA is signed unfortunately. Would you be able to sign the CLA?

**Comment by robholland** [comment]
Closing in favour of #182

**Pull request closed without merging**
